src/main/java/com/ebay/myriad/executor/MyriadExecutor.java

```diff
@@ -150,8 +151,48 @@ public void run() {
         driver.sendStatusUpdate(status);
     }
 
+    private boolean changeOwnership(String directory, String user, String group) throws IOException, InterruptedException {
+        ProcessBuilder changeOwnershipProceesBuilder = new ProcessBuilder("sudo", "/bin/chown",
```

Comment:
Can we assume chown always lives in `/bin/chown` in all distros? Could it ever be in `/usr/bin/chown` or elsewhere?